model/activist.go

```diff
@@ -296,3 +327,102 @@ id = :id`, user)
 	}
 	return user.ID, nil
 }
+
+func SuspendUser(db *sqlx.DB, userID int) error {
+	if userID == 0 {
+		return errors.New("SuspendUser: userID cannot be 0")
+	}
+	var userCount int
+	err := db.Get(&userCount, `SELECT count(*) FROM activists WHERE id = ?`, userID)
+	if err != nil {
+		return errors.Wrap(err, "failed to get user count")
+	}
+	if userCount == 0 {
+		return errors.Errorf("User with id %d does not exist", userID)
+	}
+
+	_, err = db.Exec(`UPDATE activists SET suspended = true WHERE id = ?`, userID)
```

Comment:
Related to my other comment, we could just use a field named `deleted` as a Boolean or time-stamp.

It really depends on what the point of "suspend" is. It sounds more like a penalty of some sort based on some business logic, but it's being updated by human intervention.

Reply:
"Suspend" does sound like a penalty to a certain degree. In this context, it seems we are hiding an activist from most parts of the UI (the exception being the event attendance page based on my testing). Perhaps `deleted`  or `hidden` would work better, but I'm ok with suspended.

Reply:
I'll change it to `hidden`.